udp_driver/package.xml

```diff
@@ -17,6 +17,7 @@
   <depend>libboost-system-dev</depend>
   <depend>rclcpp</depend>
   <depend>std_msgs</depend>
+  <depend>udp_msgs</depend>
```

Comment:
Since this package doesn't yet use `ament_cmake_auto`, it also needs a `find_package(udp_msgs REQUIRED)` in CMakeLists.txt.

Reply:
i added the `find_package(udp_msgs REQUIRED)` down with the other changes in the CMake, that way we could put a big IF statement around it if we wanted to separate out the executable vs just the header library